DeanerySystem/DeanerySystem.UI/Controllers/HomeController.cs

```diff
@@ -49,5 +63,46 @@ public ActionResult Contact()
 
             return View();
         }
+
+        #region SmokeTest
+        private void MockSmokeTest()
```

Comment:
Should the smoke tests be written in the Home controller?

Reply:
this could be written anywhere. it is just for the development purpose.